kotlin/samples/tictactoe/common/src/main/java/com/squareup/sample/gameworkflow/TakeTurnsRenderer.kt

```diff
@@ -15,17 +15,14 @@
  */
 package com.squareup.sample.gameworkflow
 
-import com.squareup.viewregistry.BackStackScreen
 import com.squareup.workflow.Renderer
 import com.squareup.workflow.WorkflowInput
 import com.squareup.workflow.WorkflowPool
 
-object TakeTurnsRenderer : Renderer<Turn, TakeTurnsEvent, BackStackScreen<GamePlayScreen>> {
+object TakeTurnsRenderer : Renderer<Turn, TakeTurnsEvent, GamePlayScreen> {
```

Comment:
Note that the game screens are no longer in a backstack. Never really made sense for them.

However, this also shows a weakness with the current ViewRegistry, where transition effects can only be assigned to back stack traversals. Just seems arbitrary.